.eslintrc.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
lol it was more or less auto-generated from PlatformUIBundle code base actually :)